content/pages/privacy.mdx

```diff
@@ -0,0 +1,206 @@
+---
```

Comment:
As per my conversation with Cameron, the policy can be changed to read in American English, as the meaning of the terms within will not be affected by spelling. This way the document can be compliant with SSW rules regarding US english.